test/rest-api/request/body/body-form-data.page.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <form id="sign-in-form">
```

Comment:
[idea only slightly related to the purpose of PR]:

`<form action="/sign-in" method="POST">` should do the same thing as you do manually in `<script>`, but not sure how `pageWith` handles submitting real forms

Reply:
That's a good point, but I'd rather leave the `FormData` construction to be explicit. People can then see where and how that form data is created and how it's passed to the request. It's not always you submit a form in that conventional way like you've described. I'd even say in these modern times you rarely do so.